src/neoxp/Extensions/ExpressRpcFoundStates.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Neo.IO.Json;
+
+namespace NeoExpress
+{
+    public class ExpressRpcFoundStates
+    {
+        public bool Truncated;
+        public (string key, string value)[] Results = new (string key, string value)[0];
+        public (bool hasValue, byte[]? value) FirstProof;
```

Comment:
Why do we have both `hasValue` and a nullable `value`? Seems like one or the other would suffice.